tools/integration-tests/tester/framework/network.go

```diff
@@ -124,6 +124,9 @@ func (n *Network) CreatePeer(c GoShimmerConfig) (*Peer, error) {
 	if config.SyncBeaconBroadcastInterval == 0 {
 		config.SyncBeaconBroadcastInterval = 5
 	}
+	if config.SyncBeacon {
+		config.Seed = syncBeaconSeed
+	}
```

Comment:
When creating peers via `framework.CreateNetwork`, the syncbeacon (1st peer, just like the faucet) has a predefined seed: `framework.syncBeaconSeed`.
This was lacking if you created the peers via `network.createPeer`.

Reply:
This is actually bad. Coz in the syncbeacon tests where there are several beacons, this will set all of the them to use the same seed.
So I'm getting rid of this.